### ISSUE TITLE
NTR: Anti-NR2A antibodies (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8709]
New term request Anti-NR2A antibodies (segal_080910193223): N-methyl-d-aspartate receptor subunit 2A; NMDA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000347
- label: Anti-NR2A antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:28154472
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against the N-Methyl-D-aspartate (NMDA) receptor NR2A subunit.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8709
- synonyms:
